docs/pages/components/Notice.mdx

```diff
@@ -0,0 +1,192 @@
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      Information
+    </h3>
+  }
+>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+
+<Notice
+  type="caution"
+  title={
+    <h3
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      Caution
+    </h3>
+  }
+>
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Different icon
+
+<Note>
+  The `icon` prop takes a string that corresponds to a [Cauldron
+  Icon](/components/icon). If you would like to use a custom icon, you can
+  compose the contents of the `Notice` component (see
+  <a href="/components/Notice#notice-with-composed-contents">example</a>).
+</Note>
+
+```jsx example
+<Notice type="caution" title="Dynamo!" icon="bolt">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
+
+### Notice With Composed Contents
+
+```jsx example
+<Notice>
+  <div className="Notice__title">
+    <Icon type="star" />
+    <h2
+      style={{
+        margin: 0,
+        padding: 0,
+        fontSize: 'var(--text-size-small)',
+        fontWeight: 'var(--font-weight-medium)'
+      }}
+    >
+      You are a star!
+    </h2>
+  </div>
+  <p style={{ margin: 'var(--space-half) 0 0 var(--space-half)' }}>
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at
+    porttitor sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus
+    arcu condimentum sed.
+  </p>
+</Notice>
+```
+
+### Notice (thin)
+
+Rendering a `Notice` that appears as a thin bar can be accomplished by only passing a `title` prop, or only passing a child.
+
+```jsx example
+<Notice
+  type="caution"
+  title="Caution, about to tread into dangerous territory"
+/>
+
+<Notice type="info">
+  <div
+    className="Notice__content"
+    style={{ display: 'inline-flex', gap: 'var(--space-half)', justifyContent: 'center', alignItems: 'center' }}
+  >
+  <Icon type="info-circle-alt" />
+    <span>A new update has been released!</span>
+  </div>
+</Notice>
+
+```
+
+### Notice (with HTML element properties)
+
+The `Notice` component will support the spreading of any props normally available to a `div` element. This includes
+properties such as `role`, `aria-label`, `aria-labelledby`, `id`, etc.
+
+```jsx example
+<Notice type="info" title="Alert!" role="alert">
+  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aliquam at porttitor
+  sem. Aliquam erat volutpat. Donec placerat nisl magna, et faucibus arcu
+  condimentum sed.
+</Notice>
+```
```

Comment:
We haven't typically been documenting additional html properties. I'm not opposed to doing so, but I wonder if there's a generic way we could maybe think about doing this via component props for other components?

Reply:
Yeah, I honestly don't see a reason to typically, and other component libraries take a similar approach (e.g., MUI specifically calls out in a few places they don't generally describe HTML props). 

Where I've seen exceptions made, is when a component is pretty much guaranteed to be used with certain HTML props and the usage is expected to be common enough to warrant an example.

A good example where MUI makes an exception is with its Button component, specifically usage of the `onClick` prop. I'm fine with removing it, but I saw this example as being similar to that. 

To your point, I think it would be worth calling this out elsewhere like you mentioned regardless of what we do here.